### DISCUSSION TITLE
Is there any plan to support exllama?

### DISCUSSION THREAD

**Discussion by yarray** [mudler/LocalAI#763] [Q&A] [answered]
Although llama.cpp can now support GPU via cublas, it seems that [exllama](https://github.com/turboderp/exllama) runs times faster if with a good enough GPU (3090 as an example). Is there any plan to support exllama, or in general, other loaders to load LLM?

**Comment by tmm1** [comment]
@mudler could there theoretically be a grpc backend implemented in python that used exllama directly?

**Reply by mudler** [reply]
Yes, definetly - in the same way now python embeddings could be an option as well

**Reply by fblissjr** [reply]
If there's an modular way to bring in GPTQ models via exllama, autogptq (https://github.com/PanQiWei/AutoGPTQ/), I'd love to use LocalAI - have only used other options (like ooba's openai extension) because of the GPTQ support.

**Reply by mudler** [reply]
yes now it's possible - I'm converting this as an issue to track it!

**Reply by mudler** [reply, marked as answer]
created: https://github.com/go-skynet/LocalAI/issues/796

**Reply by mudler** [reply]
JFYI: coming soon https://github.com/go-skynet/LocalAI/pull/881
